src/main/java/com/regnosys/rosetta/common/serialisation/RosettaObjectMapper.java

```diff
@@ -12,17 +18,24 @@ public class RosettaObjectMapper {
     private RosettaObjectMapper() {
     }
 
-    public static ObjectMapper getNewMinimalRosettaObjectMapper() {
-        return new AnnotationBasedObjectMapperCreator(false).create()
-                //This call is scans the classpath and can be slow. Use the AnnotationBasedObjectMapperCreator directly if you don't need it.
-                .findAndRegisterModules();
+    public static ObjectMapper getRosettaJSONMapper() {
+        return new RosettaJSONObjectMapperCreator(false).create();
     }
 
-    /**
-     * Creating new RosettaObjectMapper instances is expensive, use the singleton instance if possible.
-     */
-    public static ObjectMapper getNewRosettaObjectMapper() {
-        return getNewMinimalRosettaObjectMapper()
+    public static ObjectMapper getOptimizedRosettaJSONMapper() {
+        return getRosettaJSONMapper()
                 .registerModule(new BlackbirdModule());
     }
+
+    public static ObjectMapper getRosettaXMLMapper(final RosettaXMLConfiguration config) {
```

Comment:
I do not think that we need these methods. Unlike the JSON version, this is not a general purpose that we will use in many places. I would be happy for users of the `getRosettaXMLMapper` to just new up a `RosettaXMLObjectMapperCreator`.